### DISCUSSION TITLE
Add options to collapse friend and clan chat to the chat filter plugin

### DISCUSSION THREAD

**Discussion by SteveDevGit** [runelite/runelite#17883] [Ideas]
Very straightforward request. We have the options to collapse game and public chat messages. Can we also have this for other types of chats?

**Comment by cdfisher** [comment]
I already have a local branch that does about 80% of this - minus being a separate config option. If this is something the people who make the decisions want, I can add in the config option and get a PR opened later today.

**Comment by SteveDevGit** [comment]
@cdfisher sounds good, please tag the PR here if you do

**Comment by SteveDevGit** [comment]
@cdfisher any update on this?

**Reply by cdfisher** [reply]
I brought it up in the development channel a while back but didn't get an answer from core maintainers regarding a few of the questions I had about how they would want some of the finer details handled, so it got shelved for the time being.
